content/en/docs/reference/kubectl/conventions.md

```diff
@@ -33,6 +33,27 @@ For `kubectl run` to satisfy infrastructure as code:
 * Switch to configuration files checked into source control for features that are needed, but not expressible via `kubectl run` flags.
 
 #### Generators
+You can create the following resources using `kubectl create` with the `--dry-run -o yaml` flag:
+
+Resources you can create using kubectl run:
```

Comment:
Maybe quote "kubectl run" with backtiqs to highlight it is a command?

Reply:
ooo, u catch my mistake.

Can I change it to 
```
Resources you can create using `kubectl create` command:
```

Reply:
ooo, u catch my mistake.

Can I change it to 
```
Resources you can create using `kubectl create` command:
```

Reply:
"You can generate the following resources with `kubectl create` command:" sounds better to me.

Reply:
I update it. plz review it